custom_components/waste_collection_schedule/waste_collection_schedule/source/aw_harburg.py

```diff
@@ -0,0 +1,203 @@
+import requests
+import json
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+from bs4 import BeautifulSoup
+
+TITLE = "AW Harburg"
+DESCRIPTION = "Abfallwirtschaft Landkreis Harburg"
+URL = "https://www.landkreis-harburg.de/bauen-umwelt/abfallwirtschaft/abfallkalender/"
+
+TEST_CASES = {
+    "CityWithTwoLevels": {
+        "district_level_1": "Hanstedt",
+        "district_level_2": "Evendorf",
+    },
+    "CityWithThreeLevels": {
+        "district_level_1": "Buchholz",
+        "district_level_2": "Buchholz mit Steinbeck (ohne Reindorf)",
+        "district_level_3": "Seppenser Mühlenweg Haus-Nr. 1 / 2",
+    },
+}
+
+class Source:
+    def __init__(self, district_level_1, district_level_2, district_level_3=None):
+        self._district_level_1 = district_level_1
+        self._district_level_2 = district_level_2
+        self._district_level_3 = district_level_3
+        self._ics = ICS()
+
+    def fetch(self):
+        # Use a session to keep cookies and stuff
+        s = requests.Session()
+        
+        # Creat some fake header because for some reason people seem to believe it is bad
+        # to read public garbage collection data via a script
+        headers = {
+            'Connection': 'keep-alive',
```

Comment:
Do you really need all these headers? Please try to use only the absolut minimum.